src/test/java/com/google/api/codegen/config/GapicConfigProducerTest.java

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.config;
+
+import com.google.api.codegen.CodegenTestUtil;
+import com.google.api.codegen.ConfigProto;
+import com.google.api.tools.framework.model.Diag;
+import com.google.api.tools.framework.model.Model;
+import com.google.api.tools.framework.model.SimpleLocation;
+import com.google.api.tools.framework.model.testing.TestDataLocator;
+import com.google.common.truth.Truth;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class GapicConfigProducerTest {
```

Comment:
We already have a config generation test here: https://github.com/googleapis/toolkit/blob/master/src/test/java/com/google/api/codegen/ConfigGenerationTest.java. There are also already existing copies of `library.proto` checked into the `testsrc` directory and a baseline at https://github.com/googleapis/toolkit/blob/master/src/test/java/com/google/api/codegen/testdata/library_config.baseline.

Reply:
+1

Reply:
This is not about generation but parsing.

Reply:
Regardless, we shouldn't need to introduce another copy of library.proto and library.yaml.

Reply:
The library.proto under testsrc contains "book_from_anywhere.proto", meaning it is not a pure copy of library.proto from googleapis repo. I don't necessarily need to use library.proto, any canonical proto will work. OTOH, I see other test set defined their own testdata subfolder and add proto/yamls there. What is the best practice here in toolkit repo. I am okay with any practice.

Reply:
Switched to use myproto.proto, which is simpler and good enough for my case. PTAL.